src/config.ts

```diff
@@ -3,4 +3,5 @@ export const config = {
   CLOUDINARY_NAME: process.env.REACT_APP_CLOUDINARY_CLOUD_NAME,
   CLOUDINARY_MAX_FILE_SIZE: process.env.REACT_APP_CLOUDINARY_MAX_FILE_SIZE,
   CLOUDINARY_PREFIX: process.env.REACT_APP_CLOUDINARY_PREFIX,
+  DEFAULT_NETWORK: process.env.REACT_APP_ETH_CHAIN,
```

Comment:
we shouldn't need to have a default here. this should always be based on the current network, which comes from the provider. if for some reason we can't find that, then this just shouldn't render.